conf/evolutions/reversions/126-add-webauthn-credentials.sql

```diff
@@ -0,0 +1,10 @@
+START TRANSACTION;
+
+-- This reversion might take a while because it needs to search in all annotation layer names for '$' and replace it with ''
+do $$ begin ASSERT (select schemaVersion from webknossos.releaseInformation) = 126, 'Previous schema version mismatch'; end; $$ LANGUAGE plpgsql;
+
+DROP TABLE webknossos.webauthnCredentials;
+
+UPDATE webknossos.releaseInformation SET schemaVersion = 125;
+
```

Comment:
Just a reminder: the version numbers here need to be bumped up as the master schema version is currently 128 and maybe even higher when this pr is merged